test-snapshots/src/snapshots-output/Snapshot.List.ss

```diff
@@ -25,10 +25,10 @@
     (scm:cons 1 (scm:quote ())))
 
   (scm:define curried
-    rt:cons)
+    rt:kons)
 
   (scm:define cons2
-    (rt:cons 2))
+    (rt:kons 2))
```

Comment:
I'm guessing the reason why this is being renamed is to make it that much clearer that this is not the scheme `cons`?

Reply:
`list-cons`?

Reply:
> I'm guessing the reason why this is being renamed is to make it that much clearer that this is not the scheme cons?

Yes, and I removed the prefix `scm:` from the `chezscheme` import which introduced this name clash. I didn't want to bother doing renamed imports for this single name

`list-cons` is obviously a better name. I'll use that. Thanks!